### DISCUSSION TITLE
Web Command Admin Enabled Password?

### DISCUSSION THREAD

**Discussion by MhysTMC4** [aldostools/webMAN-MOD#715] [Q&A] [answered]
Hello. I would like to enable Admin Mode on my web browser, but I don't know the password for it. I would like to do it from my web browser instead of doing the button combo of enabling it. What is the password for it?

**Comment by aldostools** [comment, marked as answer]
The admin mode uses the same password configured in /setup.ps3 for FTP server.

If you forgot the password, use the button combo `L2`+`R2`+`△` on the PS3 to re-enable the admin mode.

The user mode is mainly intended to restrict remote users to access to settings or file operations.

This mode should be combined with other security methods like the IP filter found in /setup.ps3 and firewall rules.
